### PR TITLE
[auditd] Upgrade ECS to 8.0.0

### PR DESCRIPTION
**Pull request by r00tu53r**
Automated PR.

Upgrades ECS to 8.0.0.

### CODE REVIEW

File: packages/auditd/manifest.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ license: basic
 categories:
   - os_system
 conditions:
-  kibana.version: ^7.14.0 || ^8.0.0
+  kibana.version: ^7.16.0 || ^8.0.0
```
Comment (adriansr) [inline packages/auditd/manifest.yml:17]: I understand we shouldn't increment the minimum version unless we introduce a feature that requires a higher version

**Review by adriansr — CHANGES_REQUESTED** [review]

File: packages/auditd/changelog.yml [hunk 2]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.0.0"
```
Comment (efd6) [inline packages/auditd/changelog.yml:2]: Just confirming that this is bumped to 2 because of ppid -> parent.pid change.

**Review by efd6 — APPROVED** [review]

File: packages/auditd/changelog.yml (hunk 2 above)
Comment (r00tu53r) [reply packages/auditd/changelog.yml:2]: Yes that's right as it results in a breaking change.

**Review by P1llus — APPROVED** [review]

**Review by adriansr — APPROVED** [review]

File: packages/auditd/changelog.yml [hunk 3]
```diff
@@ -1,4 +1,9 @@
...
+  changes:
+    - description: Update to ECS 8.0
+      type: enhancement
```
Comment (andrewkroh) [inline packages/auditd/changelog.yml:5]: It's also a "breaking change". Perhaps there should be a secondary change listed with the same PR link that describes that `process.ppid` was replaced with `process.parent.pid`.

**Pull request merged**